docs/build_docs/source/flags/all_packages/index.rst

```diff
@@ -157,46 +157,136 @@ Configure the parameters of the PPP (Protein Pele Preparation)
+|             | "openff-1.2.0"       |              |                                    |
+|             |                      |              |                                    |
+|             | "openff-1.1.1"       |              |                                    |
+|             |                      |              |                                    |
+|             | "openff-1.1.0"       |              |                                    |
+|             |                      |              |                                    |
+|             | "openff-1.0.1"       |              |                                    |
+|             |                      |              |                                    |
+|             | "openff-1.0.0"       |              |                                    |
++-------------+----------------------+--------------+------------------------------------+
 
-- **core**: Atomnumber of the atom that will be included as part of the rigid core. Default=None
+PlopRotTemp
+++++++++++++
 
-- **maxtorsion**: Maximum number of rotamers per flexible sidechain. Default: 4
+To continue using PlopRotTemp, you do not need to make any changes to your YAML file, previously existing flags are still
+available:
 
-- **n**: Maximum number of flexible sidechains in a molecule, Default: None
+    - **gridres**: Resolution of the rotamers when sampling them by the Side Chain prediction algorithm. Default=10 degrees
 
-- **mae_lig**: Mae file to extract the cuantum charges from. Default: None
+    - **core**: List of PDB atom names that will be included as part of the rigid core. In case it is not specified, the algorithm will pick up a set of non-rotatable atoms centered in the molecular structure. Default=None
 
-- **template**: External forcefield templaters
+    - **exclude_terminal_rotamers**: Exclude terminal rotamers during parametrization of non standard molecules if they belong to a small terminal group. Default=True
 
-- **rotamers**: External rotamer libraries
+    - **mae_lig**: External MAE file with quantum charges generated with Schrödinger suite. When supplied, any charge calculated internally in the platform will be replaced by the charges from this file. Default=None
 
-- **skip_ligand_prep**: Skip preparation of that resiude. This could be usefull to bypass problems with PlopRotTemp when creating the ligand parameters.
+    - **maxtorsion**: Maximum number of rotamers per flexible side chain. Default=4
+
+    - **n**: Maximum number of flexible side chains in a molecule. Default=None
 
 
 ..  code-block:: yaml
 
-  gridres: 10
-  core: -1
-  maxtorsion: 4
-  n: 5
-  mae_lig: "/home/dsoler/lig.mae"
+    solvent: "OBC"
+    maxtorsion: 4
+    n: 5
+    mae_lig: "/home/dsoler/lig.mae"
+    gridres: 10
+
+Peleffy
+++++++++++
+
+You can use the following parameters to control the way peleffy will parametrize non-standard molecules for you:
```

Comment:
Here I am missing the part where we tell the users how they can use peleffy. Now we have a flag to ask for peleffy instead of plop, right? So, I would say: In order to use peleffy, you first need to add the following flag [no idea about the flag but let's imagine it is use_peleffy] use_peleffy and set it to true.